titan/params/partnership.yml

```diff
@@ -179,7 +179,8 @@ partnership:
     type: sub-dict
     keys:
       - bond_types
-    description: "Duration of relationships given bond type"
+      - races
+    description: "Duration of relationships given agent race and partnership bond type"
```

Comment:
why key this off of race? it doesn't seem to have affected any of the settings

Reply:
It only affects NYC at this point, but should affect others going forward as well.